File: dal-dao-gen/dal_ftp/63/java/Entity/Test1.java
package com.ctrip.dal.test.test4;


import com.ctrip.platform.dal.dao.DalPojo;

public class Test1 implements DalPojo {
	private Integer htlOrderDetailId;
	private Integer orderId;
	private String orderType;
	private BigDecimal amount;
	private BigDecimal serviceFee;
	private BigDecimal rebate;
	private String isInbatch;
	private Integer accCheckId;
	private Timestamp creatTime;
	private Timestamp inbatchTime;
	private Timestamp lastModifyTime;
	private Integer accountId;
	private Integer subAccountID;
	private Integer rid;
	private Timestamp rCTime;
	private Integer rCQuantity;
	public Integer getHtlOrderDetailId() {
		return htlOrderDetailId;
	}

	public void setHtlOrderDetailId(Integer htlOrderDetailId) {
		this.htlOrderDetailId = htlOrderDetailId;
	}

	public Integer getOrderId() {
		return orderId;
	}

	public void setOrderId(Integer orderId) {
		this.orderId = orderId;
	}

	public String getOrderType() {
		return orderType;
	}

	public void setOrderType(String orderType) {
		this.orderType = orderType;
	}

	public BigDecimal getAmount() {
		return amount;
	}

	public void setAmount(BigDecimal amount) {
		this.amount = amount;
	}

	public BigDecimal getServiceFee() {
		return serviceFee;
	}

	public void setServiceFee(BigDecimal serviceFee) {
		this.serviceFee = serviceFee;
	}

	public BigDecimal getRebate() {
		return rebate;
	}

	public void setRebate(BigDecimal rebate) {
		this.rebate = rebate;
	}

	public String getIsInbatch() {
		return isInbatch;
	}

	public void setIsInbatch(String isInbatch) {
		this.isInbatch = isInbatch;
	}

	public Integer getAccCheckId() {
		return accCheckId;
	}

	public void setAccCheckId(Integer accCheckId) {
		this.accCheckId = accCheckId;
	}

	public Timestamp getCreatTime() {
		return creatTime;
	}

	public void setCreatTime(Timestamp creatTime) {
		this.creatTime = creatTime;
	}

	public Timestamp getInbatchTime() {
		return inbatchTime;
	}

	public void setInbatchTime(Timestamp inbatchTime) {
		this.inbatchTime = inbatchTime;
	}

	public Timestamp getLastModifyTime() {
		return lastModifyTime;
	}

	public void setLastModifyTime(Timestamp lastModifyTime) {
		this.lastModifyTime = lastModifyTime;
	}

	public Integer getAccountId() {
		return accountId;
	}

	public void setAccountId(Integer accountId) {
		this.accountId = accountId;
	}

	public Integer getSubAccountID() {
		return subAccountID;
	}

	public void setSubAccountID(Integer subAccountID) {
		this.subAccountID = subAccountID;
	}

	public Integer getRid() {
		return rid;
	}

	public void setRid(Integer rid) {
		this.rid = rid;
	}

	public Timestamp getRCTime() {
		return rCTime;
	}

	public void setRCTime(Timestamp rCTime) {
		this.rCTime = rCTime;
	}

	public Integer getRCQuantity() {
		return rCQuantity;
	}

	public void setRCQuantity(Integer rCQuantity) {
		this.rCQuantity = rCQuantity;
	}

}

File: dal-dao-gen/dal_ftp/63/cs/Entity/Test1.cs
using System;
using Arch.Data.Orm;

namespace com.ctrip.dal.test.test4.Entity.DataModel
{
    /// <summary>
    /// test1
    /// </summary>
    [Serializable]
    [Table(Name = "test1")]
    public partial class Test1
    {
        /// <summary>
        /// 获取或设置HtlOrderDetailId
        /// </summary>
        [Column(Name = "HtlOrderDetailId",Length=10),ID]
        public int HtlOrderDetailId { get; set; }
        /// <summary>
        /// 获取或设置OrderId
        /// </summary>
        [Column(Name = "OrderId",Length=10)]
        public int? OrderId { get; set; }
        /// <summary>
        /// 获取或设置OrderType
        /// </summary>
        [Column(Name = "OrderType",Length=1)]
        public char? OrderType { get; set; }
        /// <summary>
        /// 获取或设置Amount
        /// </summary>
        [Column(Name = "Amount",Length=19)]
        public decimal? Amount { get; set; }
        /// <summary>
        /// 获取或设置ServiceFee
        /// </summary>
        [Column(Name = "ServiceFee",Length=19)]
        public decimal? ServiceFee { get; set; }
        /// <summary>
        /// 获取或设置Rebate
        /// </summary>
        [Column(Name = "Rebate",Length=19)]
        public decimal? Rebate { get; set; }
        /// <summary>
        /// 获取或设置IsInbatch
        /// </summary>
        [Column(Name = "IsInbatch",Length=1)]
        public char? IsInbatch { get; set; }
        /// <summary>
        /// 获取或设置AccCheckId
        /// </summary>
        [Column(Name = "AccCheckId",Length=10)]
        public int? AccCheckId { get; set; }
        /// <summary>
        /// 获取或设置CreatTime
        /// </summary>
        [Column(Name = "CreatTime",Length=23)]
        public DateTime? CreatTime { get; set; }
        /// <summary>
        /// 获取或设置InbatchTime
        /// </summary>
        [Column(Name = "InbatchTime",Length=23)]
        public DateTime? InbatchTime { get; set; }
        /// <summary>
        /// 获取或设置LastModifyTime
        /// </summary>
        [Column(Name = "LastModifyTime",Length=23)]
        public DateTime? LastModifyTime { get; set; }
        /// <summary>
        /// 获取或设置AccountId
        /// </summary>
        [Column(Name = "AccountId",Length=10)]
        public int? AccountId { get; set; }
        /// <summary>
        /// 获取或设置SubAccountID
        /// </summary>
        [Column(Name = "SubAccountID",Length=10)]
        public int? SubAccountID { get; set; }
        /// <summary>
        /// 获取或设置Rid
        /// </summary>
        [Column(Name = "Rid",Length=10)]
        public int? Rid { get; set; }
        /// <summary>
        /// 获取或设置RCTime
        /// </summary>
        [Column(Name = "RCTime",Length=23)]
        public DateTime? RCTime { get; set; }
        /// <summary>
        /// 获取或设置RCQuantity
        /// </summary>
        [Column(Name = "RCQuantity",Length=10)]
        public int? RCQuantity { get; set; }
    }
}